tasks/install.yml

```diff
@@ -1,48 +1,35 @@
 ---
-- name: Add system user
-  user:
-    name: "{{ react_system_user }}"
-    shell: /bin/bash
-    group: "{{ react_system_group }}"
-    append: yes
-    createhome: yes
-
-- name: Add yarn apt key
-  shell: curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-  become: True
-  become_user: root
-  args:
-    warn: no
-
-- name: Add yarn to sources
-  shell: echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-  become: True
-  become_user: root
-
-- name: Download node
-  shell: curl -sL https://deb.nodesource.com/setup_{{ react_node_version }} | sudo -E bash -
-  become: True
-  become_user: root
-  args:
-    warn: no
-
-- name: Update apt cache
-  apt:
-    update_cache: "yes"
+- name: Install NVM
```

Comment:
We might skip installing NVM and make nvm on the deployment a prerequisite for the role to run.